test/metabase/api/search_test.clj

```diff
@@ -52,7 +53,9 @@
    {:name name}
    (apply array-map kvs)))
 
-(def ^:private test-collection (make-result "collection test collection", :model "collection", :collection {:id true, :name true}, :updated_at false))
+(def ^:private test-collection (make-result "collection test collection", :model "collection",
+                                            :collection {:id true, :name true :type nil},
```

Comment:
This is really not important but it makes me itchy to separate the first and second key-value pair with a comma but not the second and third. I'm only bringing it up because I don't want future new hires to infer that this is the idiomatic way to format stuff or people new to Clojure think there's some sort of special reason there's one comma there instead of two or zero